kornia/contrib/models/efficient_vit/nn/norm.py

```diff
@@ -0,0 +1,37 @@
+# EfficientViT: Multi-Scale Linear Attention for High-Resolution Dense Prediction
+# Han Cai, Junyan Li, Muyan Hu, Chuang Gan, Song Han
+# International Conference on Computer Vision (ICCV), 2023
+from __future__ import annotations
+
+from typing import Any, Optional
+
+import torch
+from torch import nn
+
+from kornia.contrib.models.efficient_vit.utils import build_kwargs_from_config
+
+
+class LayerNorm2d(nn.LayerNorm):
```

Comment:
by code cov, looks like it isn't used

Reply:
defined below in the REGISTERED_NORM_DICT

Reply:
but if it wasn't executed by the tests, it probably wasn't used anywhere else?